### PR TITLE
adding for docs for insecureEdgeTerminatePolicy in reencrypt and passthrough routes 

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
update to the docs for origin PR:[11953](https://github.com/openshift/origin/pull/11953)

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@JacobTanenbaum Is this for 3.4? Thanks!

**Comment by JacobTanenbaum** [comment]
3.5

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -683,6 +689,11 @@ validate the endpoint certificate, securing the connection from the router to
 the destination. This field is required, but only for re-encryption.
 ====
 
+[NOTE]
+====
+Reencrypt routes can have an insecureEdgeTerminationPolicy the same as edge-terminated routes.
```
Comment (knobunc) [inline architecture/core_concepts/routes.adoc]: Reencrypt routes can have an insecureEdgeTerminationPolicy with all of the same values as for edge-terminated routes.

**Review by knobunc — APPROVED** [review]
LGTM

**Comment by JacobTanenbaum** [comment]
@ahardin-rh @knobunc the code associated with this documentation has merged

**Comment by ahardin-rh** [comment]
@JacobTanenbaum Thank you!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Core Concepts -> Routes]
|Added details about `insecureEdgeTerminatePolicy` in regard to reencrypt and passthrough routes.
%

**Pull request merged**